src/pyedb/dotnet/edb_core/edb_data/padstacks_data.py

```diff
@@ -499,7 +499,7 @@ def pad_parameters(self) -> dict:
             self._ppadstack._pedb._edb.Definition.PadType.RegularPad,
             self._ppadstack._pedb._edb.Definition.PadType.AntiPad,
             self._ppadstack._pedb._edb.Definition.PadType.ThermalPad,
-            self._ppadstack._pedb._edb.Definition.PadType.Hole,
+            # self._ppadstack._pedb._edb.Definition.PadType.Hole,
```

Comment:
Is that a miss or on purpose ? If on purpose can you comment why and resolve this comment ?

Reply:
@hui-zhou-a can you please check this comment so we can merge the PR ?

Reply:
Added comments.
# This property doesn't appear in UI. It is unclear what it is used for. Suppressing this property for now.